workflow/rules/read_clipping.smk

```diff
@@ -22,7 +22,7 @@ rule samtools_sort:
 
 rule bed_to_bedpe:
     input:
-        check_bed_for_URL(config["preprocessing"]["amplicon-primers"]),
+        check_bed_for_URL(config["preprocessing"]["amplicon-primers"]["illumina"]),
```

Comment:
What if no primers are used? This will fail with a key error then.

Reply:
illumina + amplicon exclusive